Intro-JS/homework/homework.js

```diff
@@ -27,79 +27,116 @@ const newModulo = 21 % 5 === null ;
 
 function returnString(str) {
   //simply return the string provided: str
+  const push = str;
```

Comment:
no need for this. only declare return str (since it is an argument for the returnString fuction)